recipes/PyMaxflow/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "PyMaxflow" %}
+{% set version = "1.2.13" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/PyMaxflow-{{ version }}.tar.gz
+  sha256: 947b517ae58a348b17f9599caefc53d58989b9ecaf11f97b4c74208aa3bb1f22
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - cython
+    - numpy
+    - pip
+    - python
+  run:
+    - python
+    - {{ pin_compatible('numpy') }}
+
+test:
+  imports:
+    - maxflow
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/pmneila/PyMaxflow
+  summary: A mincut/maxflow package for Python
+  license: GPL-3.0-or-later
+  license_file: LICENSE
```

Comment:
For posterity, the licensing of the Python wrapper was confirmed upstream here: https://github.com/pmneila/PyMaxflow/issues/53.